approve-pending-signatures.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+Small script that can be used to approve all the pending signatures
+To be used with extreme caution
+*/
+
+import org.jenkinsci.plugins.scriptsecurity.scripts.*
+ScriptApproval sa = ScriptApproval.get();
+
+println "Signatures pending Approval..."
+for (ScriptApproval.PendingSignature pending : sa.getPendingSignatures()) {
+        println "Pending approval : " + pending.signature
+}
+
+
+println "Starting cleanup..."
+sa = ScriptApproval.get();
+
+
+
+while(sa.getPendingSignatures().size()>0) {
+  try{
+        item= sa.getPendingSignatures()[0]
+        println item.signature
```

Comment:
@MCardenasBlanco I'd suggests that we add a warning and prevent approving blacklisted signatures here:

```suggestion
        if (!StaticWhitelist.isBlacklisted(item. signature)) {
             println "[WARNING] Not approving " + item. signature + " because it is blacklisted"
        }
        println item.signature
```